packages/app-bridge-theme/src/types/Guideline.ts

```diff
@@ -11,41 +11,48 @@ export enum LinkSettingsIconPosition {
     Left = 'LEFT',
 }
 
-export interface GuidelineCoverPage {
+interface CoverPageBase {
     id(): number;
     title(language?: string): string;
     isPublished(): boolean;
     isHiddenInNavigation(): boolean;
     url(language?: string): string;
     type: 'cover-page';
 }
+export interface CoverPageNavigationItem extends CoverPageBase {}
+export interface CoverPage extends CoverPageBase {}
 
-export interface GuidelineDocumentGroup {
+interface DocumentGroupBase {
     id(): number;
     title(language?: string): string;
-    children(): (GuidelineDocument | GuidelineDocumentLibrary | GuidelineDocumentLink)[];
+    children(): (DocumentNavigationItem | DocumentLibraryNavigationItem | DocumentLinkNavigationItem)[];
     type: 'document-group';
 }
+export interface DocumentGroupNavigationItem extends DocumentGroupBase {}
```

Comment:
IMO we don’t need the `DocumentGroupBase` for now. Once we need a `DocumentGroup` we can easily add it. Wdyt?

Reply:
well, we do need it. Themes already use the current `GuidelineDocumentGroup`

Reply:
@anxobotana we only use the navigation item. But my point is only that we don’t need the base interface when only one interface uses it. I would just reduce the complexity and keep it simple.